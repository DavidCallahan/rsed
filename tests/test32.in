473.astar
483.xalancbmk," -DSPEC_CPU_LINUX"

"Peak Optimization Flags"
400.perlbench," -O3"," -fno-strict-aliasing"
401.bzip2," -O3"," -fno-strict-aliasing"
