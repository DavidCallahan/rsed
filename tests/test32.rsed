skip past required "Peak Optimization Flags"
required "^[^,]+,(.*)$"
$temp = replace '[",\' ]+'g with ' ' in $1
print "peak_opts=" quote("'", $temp)
