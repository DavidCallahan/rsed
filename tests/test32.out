peak_opts=' -O3 -fno-strict-aliasing '
